build.gradle

```diff
@@ -2500,6 +2502,7 @@ project(':tools') {
     testImplementation libs.apachedsLdifPartition
 
     testRuntimeOnly libs.junitPlatformLanucher
+    testRuntimeOnly libs.hamcrest
```

Comment:
One wrinkle here is that org.apache.kafka.streams.integration.utils.IntegrationTestUtils relies on Hamcrest, so I add it to the tools module.